torchtext/experimental/vocab.py

```diff
@@ -77,6 +77,25 @@ def vocab_from_file(file_object, min_freq=1, unk_token='<unk>', num_cpus=4):
     return Vocab(vocab_obj)
 
 
+def build_vocab_from_iterator(iterator, min_freq=1, unk_token='<unk>'):
```

Comment:
I'd want us to remove "unk_token" with a more generic solution in a follow-up PR so that we can also support Vocabs that fail if a key isn't present.

Reply:
Will do it in the follow-up PR for the unknown token.